components/steps/Status.tsx

```diff
@@ -156,7 +156,8 @@ const StatusStep: React.FunctionComponent<Props> = props => {
                 </StepButton>
               )}
             </Step>
-            <Step disabled={profileStage == 4}>
+            {/* 3 and 4 is awaiting results and received results, respectively*/}
```

Comment:
wait I think we are leaving these as is because the confirmation page still has the t-shirt size